### PR TITLE
Added social_media module and enabled it for article content type via field.

### PR DESCRIPTION
**Pull request by Vinay15**

### CODE REVIEW

File: modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_form_display.node.article.default.yml [hunk 1]
```diff
@@ -7,12 +7,14 @@ dependencies:
...
     - field.field.node.article.field_meta_tags
     - field.field.node.article.field_short_title
+    - field.field.node.article.field_social_sharing
     - field.field.node.article.field_subhead
     - field.field.node.article.field_summary
     - field.field.node.article.field_tags
     - field.field.node.article.field_thumbnail
     - field.field.node.article.field_yoast_seo
     - node.type.article
+    - workflows.workflow.default
```
Comment (vaibhavjain-in) [inline modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_form_display.node.article.default.yml]: Do we need it ?

File: modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_form_display.node.article.default.yml [hunk 2]
```diff
@@ -267,6 +277,7 @@ content:
       match_operator: CONTAINS
       size: 60
       placeholder: ''
+      match_limit: 10
```
Comment (vaibhavjain-in) [inline modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_form_display.node.article.default.yml:279]: What is this change about ?

File: modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_view_display.node.article.default.yml [hunk 3]
```diff
@@ -6,14 +6,17 @@ dependencies:
     - field.field.node.article.field_content
     - field.field.node.article.field_meta_tags
     - field.field.node.article.field_short_title
+    - field.field.node.article.field_social_sharing
     - field.field.node.article.field_subhead
     - field.field.node.article.field_summary
     - field.field.node.article.field_tags
     - field.field.node.article.field_thumbnail
+    - field.field.node.article.field_yoast_seo
```
Comment (vaibhavjain-in) [inline modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_view_display.node.article.default.yml:14]: Again, is this correct ?

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@Vinay15 few config changes should not be in place. Can you please verify they are installed correctly ?

File: modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_form_display.node.article.default.yml (hunk 1 above)
Comment (Vinay15) [reply modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_form_display.node.article.default.yml]: This comes by default from the content_moderation module in core which defines the moderation workflow for Article and Basic page content types and we explicitly remove it every time.

File: modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_form_display.node.article.default.yml (hunk 2 above)
Comment (Vinay15) [reply modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_form_display.node.article.default.yml:279]: This is a change introduced in core: https://www.drupal.org/node/3024684

File: modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_view_display.node.article.default.yml (hunk 3 above)
Comment (Vinay15) [reply modules/ezcontent_node/modules/ezcontent_article/config/install/core.entity_view_display.node.article.default.yml:14]: This is added by the yoast_seo module and we remove it. Please help me understand why we need to do this.

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Pull request merged**